oci/private/tarball.bzl

```diff
@@ -87,7 +87,9 @@ def _tarball_impl(ctx):
 
     tarball = _tar_tarball(ctx, blobs, manifest)
 
-    return DefaultInfo(files = depset([tarball]))
```

Comment:
nit: revert

Reply:
Oops, I turned on "Require conversation resolution before merging"  so that doesn't happen again.